.gitignore

```diff
@@ -8,22 +8,18 @@
 # Crash log files
 crash.log
 
-# Ignore any .tfvars files that are generated automatically for each Terraform run. Most
-# .tfvars files are managed as part of configuration and so should be included in
-# version control.
-#
-# example.tfvars
-
 # Ignore override files as they are usually used to override resources locally and so
 # are not checked in
 override.tf
 override.tf.json
 *_override.tf
 *_override.tf.json
 
-# Include override files you do wish to add to version control using negated pattern
-#
-# !example_override.tf
+# OSX index files
+**/.DS_Store
+
+# vscode config
+.vscode/
 
```

Comment:
```suggestion
```

This sort of thing should go into your [global `gitignore` file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
Makes sense, I agree